tests/testthat/tests_subsetting.R

```diff
@@ -7,8 +7,28 @@ test_that("Entered Data", {
   area.column <- "STATECODE"
   w.use.sub <- subset_wuse(w.use, data.elements, area.column, areas)
   
-  expect_equal(ncol(w.use.sub), 4)
+  expect_equal(ncol(w.use.sub), 5)
   expect_equal(nrow(w.use.sub), 18)
   cklist <- c("YEAR",area.column,"PS.TOPop","PS.SWPop")
```

Comment:
OK I see the error here. So after correcting this, do I then do a completely new commit?